src/plugins/data/public/index.ts

```diff
@@ -527,3 +527,6 @@ export {
   DataSourceGroup,
   DataSourceOption,
 } from './data_sources/datasource_selector';
+
+export { SuggestionsComponent } from './ui';
```

Comment:
might be related to another PR. but for this could this be just apart of the ui service: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/830664c9298fec49ef73f50596089e7bfaab4fde/src/plugins/data/public/ui/ui_service.ts#L64

if it's not already accessible?